types/components/Carousel.d.ts

```diff
@@ -7,22 +7,24 @@ import { BsPrefixComponent } from './helpers';
 
 export interface CarouselProps {
   bsPrefix?: string;
-  slide?: boolean;
+  animate?: boolean;
```

Comment:
I guess it should be renamed to `animate` in component code too

Reply:
oh, oops. cruft on my part. the upstream has the class as just "slide", so i figured we'd keep the same naming here.

Reply:
```suggestion
  slide?: boolean;
```